### ISSUE TITLE
[FEATURE] <Decide color palette for the website>

### ISSUE THREAD

**Issue by Preet-Sojitra** [ykdojo/defaang#82]
### Description

I think we should decide the color palette for the website, so that people who working on UI make UI using those colors.
Also if we have fixed color palette then different different people who are working on UI won't come with the own color palettes.
So let's asks contributors to come up with their own color palette and decide one good color palette out from it.

Well there are many tools available to generate color palette for the website. We can also use them

What are your thoughts on this @ykdojo or do you have any preferred color scheme to move forward?

**Comment by ykdojo** [comment]
I think it might be helpful if you could suggest a few different options both for light and dark themes?

We have a little bit of that discussion started already here: https://github.com/ykdojo/defaang/issues/14#issuecomment-1214188458

**Comment by Preet-Sojitra** [comment]
Well these are some of the palettes generated by [Huemint](https://huemint.com/). 
Light theme color palette:
- https://huemint.com/website-magazine/#palette=fffdf9-fe845a-091a38-3c4f9a
- https://huemint.com/website-magazine/#palette=fefefe-f8d799-18160d-d8252c
- https://huemint.com/website-magazine/#palette=e9e7e0-fcb77c-0c100f-8d4fb3
- https://huemint.com/website-magazine/#palette=fffffd-77bbdb-081430-44a2d2
- https://huemint.com/website-magazine/#palette=fffcf7-fec4a1-211824-477df6
- https://huemint.com/website-magazine/#palette=8fca42-fbfeff-152432-c34caa
- https://huemint.com/website-magazine/#palette=f8fcfa-ecbb94-0e0629-6f3d48
- https://huemint.com/website-2/#palette=cec586-1d385a-bd2229-342b11

Dark theme color palette:
- https://huemint.com/website-2/#palette=2a2c2a-d3cdd1-eb9166-38bbce
- https://huemint.com/website-2/#palette=090605-dcd4d4-c34a00-2896c6
- https://huemint.com/website-2/#palette=062a43-edc346-dc6541-52b1c6
- https://huemint.com/website-2/#palette=16232b-4dd3b2-c1a333-727343
- https://huemint.com/website-2/#palette=1b1b2d-53f191-de595b-d2a55f

**Comment by ykdojo** [comment]
Any 2-3 of them that you'd recommend?

**Comment by Preet-Sojitra** [comment]
I will filter it out and let you know soon 👍

**Comment by Preet-Sojitra** [comment]
For light would, I would recommend 4,5 and 6

And for Dark mode I would recommend
1 and 2

**Comment by jideabdqudus** [comment]
For light mode, I'll recommend 2,4, 5
I think dark mode, is just inverting the light mode. So depending on the choice of light mode, we can decide on the darkmode theme.

**Comment by ykdojo** [comment]
4 looks pretty good for lightmode. I don't want to be the only one making the decision though - maybe we can wait for other people to pitch in their opinions.

For dark mode, I'm not sure yet...

**Comment by AdityaPainuli** [comment]
I would recommend doing some more searches for dark mode as most of the people might use dark mode only from all of the above I think 1st one for the dark mode is best compared to all of them but we should try to find something better for the dark mode.

**Comment by Dhruv-Gajjar** [comment]
I think 4 is pretty good for light mode 👍🏻.

**Comment by Halim-Shams** [comment]
I recommend 4 for light mode and 1 for dark mode. Because there is more relativity between those two Modes.

**Comment by yaa-chi** [comment]
@Preet-Sojitra thanks a bunch for the colors! They look amazing. And yes I do love 4 for the light mode. Because it also has elements of dark mode within it. I'll send a link to show you guys what I mean😉

**Comment by yaa-chi** [comment]
Here's a link to the color palette chosen for light mode and a re-arrangement for dark mode. I'll design a simple here section to show you how it will look in action.

the link: https://www.figma.com/file/BMHpagkuDfPNskxak5gQqA/Brand-Guide?node-id=0%3A1

**Comment by ykdojo** [comment]
@yaa-chi thank you! Looks really cool.

Any chance you could put the dark mode one in the same format as [Option 4 from above](https://huemint.com/website-magazine/#palette=fffffd-77bbdb-081430-44a2d2) so we can get a better idea about what it might look like?

**Comment by yaa-chi** [comment]
> @yaa-chi thank you! Looks really cool.
> 
> Any chance you could put the dark mode one in the same format as [Option 4 from above](https://huemint.com/website-magazine/#palette=fffffd-77bbdb-081430-44a2d2) so we can get a better idea about what it might look like?

Thanksss!! 🤩🏹And yes, I'm currently designing a demo for you guys to check it out as well!

**Comment by yaa-chi** [comment]
@ykdojo Have a look at a little demo I just designed. Had to do it this morning.

And everyone as well, do let me know what you think🙌🏾🎯

https://www.figma.com/file/BMHpagkuDfPNskxak5gQqA/Brand-Guide?node-id=0%3A1

**Comment by Preet-Sojitra** [comment]
It's looking good 😍

**Comment by yaa-chi** [comment]
Thank youuu @Preet-Sojitra 😍... This particular dark mode color palette was gotten from the one that everyone loved. The light mode that you suggested!

**Comment by ykdojo** [comment]
@yaa-chi I like it!

I'm wondering if you could make the contract slightly higher though? Maybe with a slightly brighter color on the text.

I'm thinking of situations like older screens, less bright screens, people looking at it on their phone in a bright room, etc.

**Comment by yaa-chi** [comment]
> @yaa-chi I like it!
> 
> I'm wondering if you could make the contract slightly higher though? Maybe with a slightly brighter color on the text.
> 
> I'm thinking of situations like older screens, less bright screens, people looking at it on their phone in a bright room, etc.

Thanksss! 🤩

Sure, absolutely will wok on the contrast asap!

**Comment by yaa-chi** [comment]
Hi @ykdojo I have updated the color contrast. Do let me know what you think so that we can proceed asap🤩

**Comment by ykdojo** [comment]
Hmm maybe even more contrast... what do you think of using white text?

**Comment by ykdojo** [comment]
I'm looking at Tailwind's website, for example
<img width="1494" alt="image" src="https://user-images.githubusercontent.com/1811651/185449971-d9076bda-518e-4530-8aed-98a442d8acc5.png">

**Comment by yaa-chi** [comment]
@ykdojo yeahhh, they used white color for the text and grey for the subtitles and message plus the navigation bar. Do you want that/

**Comment by ykdojo** [comment]
Yes! I think something like that would work well for better readability. I'm starting to notice that GitHub has the same type of color scheme too - white for the main text and grey for other things.

**Comment by yaa-chi** [comment]
Awesome, I'm gonna implement those color schemes asap!

**Comment by anaskhan28** [comment]
looks good https://www.figma.com/file/BMHpagkuDfPNskxak5gQqA/Brand-Guide?node-id=0%3A1

**Comment by veganaize** [comment]
How will this palette be preserved, after the issue / PR is closed?  Should a `DESIGN.md` or similar be created?

**Comment by olamide203** [comment]
The color palettes could be added to `theme.colors` in `tailwind.config.js` since tailwind is being used for the UI

**Comment by olamide203** [comment]
Hi @ykdojo can I add the already selected color palettes to the `tailwind.config.js`

**Comment by ykdojo** [comment]
@olamide203 yes that'll be great. Would it correctly change the colors in the Tailwind UI components?

**Comment by olamide203** [comment]
we could either extend the default color palette or completely relpace it with the new one. I think extending the default then slowly applying the colors to the UI is better.
